indexer/editable_map_object.cpp

```diff
@@ -1,3 +1,5 @@
+#include "search/v2/postcodes_matcher.hpp"
```

Comment:
And specification of search after indexer in the *.pro file doesn't help, it'll just delay the problem - search heavily depends on indexer and this dependency order is wrong - in all *.pro files search must go before indexer.